infrastructure/prod.tfvars

```diff
@@ -8,5 +8,6 @@ save_claim_state_enabled = "true"
 doc_assembly_api_url = "http://dg-docassembly-prod.service.core-compute-prod.internal"
 milo_csv_schedule = "0 0 8 * * ?"
 claim_stayed_schedule = "0 0 16 ? * MON-FRI *"
+claim_reissue_schedule = "0 0 8,16 ? * MON-FRI *"
```

Comment:
I recommend a more frequent schedule. Hourly maybe.

Reply:
OK..let me post in the team channel and lets discuss what would be the best